composer.json

```diff
@@ -10,8 +10,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
-        "psr/log": "^1.0",
-        "webmozart/path-util": "^2.3"
+        "psr/log": "^1.0"
```

Comment:
needs to be in `require`

Reply:
I have just removed it. Should I also move the logger to the dev dependencies?

Reply:
sorry, i meant, symfony/filesystem needs to be in `require`, assuming that this library used the `Path::` function

Reply:
Ah makes sense, I'll do that.